File: Baekjoon/Greedy/1758.py
# https://www.acmicpc.net/problem/1758

import sys
input = sys.stdin.readline

n = int(input())
arr = sorted([int(input()) for _ in range(n)], reverse = True)
result = 0

for i in range(n):
    if arr[i] - i > 0:
        result += arr[i] - i

print(result)

File: Baekjoon/Greedy/1758.cpp
// https://www.acmicpc.net/problem/1758

#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main() {
    int n;
    cin >> n;
    vector<int> a(n);
    for (int i = 0; i < n; i++) {
        cin >> a[i];
    }

    sort(a.begin(), a.end(), greater<int>());

    long long ans = 0;
    for (int i = 0; i < n; i++) {
        if (a[i] - i > 0) {
            ans += a[i] - i;
        }
    }

    cout << ans << '\n';

    return 0;
}